Codeinwp/otter-blocks
# fix: explicit object ID on dynamic tags Codeinwp/neve-pro-addon#2045

References Codeinwp/neve-pro-addon#2045.


### Summary

I've added explicit object ID passing for dynamic content related to posts to use data from the parent post.
Previously it would use the data of the post where defined, but when using dynamic tags inside custom layouts you would want to display data related to the post where you add the custom layout.

### Screenshots 

----

### Test instructions

1. Create a custom layout with dynamic tags
2. Include the layout inside a post or a page
3. Check that the dynamic tags display the data related to the post that is being displayed. Eg. Post Title
4. Check that on regular posts the dynamic tags display the same information as before.



---- 

### Checklist before the final review

- [x] It loads additional script in the front end only if required.
